core/math/geometry.cpp

```diff
@@ -514,7 +514,7 @@ static inline void _build_faces(uint8_t ***p_cell_status, int x, int y, int z, i
 		Vector3(1,1,1),
 	};
 */
-#define vert(m_idx) Vector3((m_idx & 4) >> 2, (m_idx & 2) >> 1, m_idx & 1)
+#define vert(m_idx) Vector3(((m_idx) & 4) >> 2, ((m_idx) & 2) >> 1, (m_idx) & 1)
```

Comment:
Apparently after this change, `clang-format` wants things formatted differently:
```diff
--- a/core/math/geometry.cpp	2019-01-30 19:51:42.607631131 +0000
+++ b/core/math/geometry.cpp	2019-01-30 19:51:52.104321095 +0000
@@ -514,7 +514,7 @@
 		Vector3(1,1,1),
 	};
 */
-#define vert(m_idx) Vector3(((m_idx) & 4) >> 2, ((m_idx) & 2) >> 1, (m_idx) & 1)
+#define vert(m_idx) Vector3(((m_idx)&4) >> 2, ((m_idx)&2) >> 1, (m_idx)&1)
 
 	static const uint8_t indices[6][4] = {
 		{ 7, 6, 4, 5 },

```

Seems bogus to me, but if it's the only occurrence, it's probably not worth fighting `clang-format` over it.